Werewolf for Telegram/Werewolf Node/Program.cs

```diff
@@ -380,23 +380,37 @@ public static void RemoveGame(Werewolf werewolf)
             }
         }
 
-        internal static async Task<Telegram.Bot.Types.Message> Send(string message, long id, bool clearKeyboard = false, InlineKeyboardMarkup customMenu = null, Werewolf game = null, bool notify = false, bool preview = false)
+        internal static async Task<Telegram.Bot.Types.Message> Send(string message, long id, bool clearKeyboard = false, InlineKeyboardMarkup customMenu = null, Werewolf game = null, bool notify = false, bool preview = false, bool isPlayerDM = false)
         {
             //MessagesSent++;
             //message = message.FormatHTML();
             //message = message.Replace("`",@"\`");
+
+            // Try to load GroupTopicId from the database, only if it is not player dm
+            int? messageThreadId = null;
+            if(!isPlayerDM)
+                using (var db = new WWContext())
+                {
+                    var group = db.Groups.FirstOrDefault(g => g.GroupId == id);
+                    if (group?.GroupTopicId != null)
+                    {
+                        messageThreadId = group.GroupTopicId;
+                    }
+                }
+            
+
```

Comment:
You're opening a database connection to get the topic ID *every time* a game message is supposed to be sent.
This is expensive and totally avoidable. The only – questionable – "advantage" that this brings is that the `/setgrouptopic` command becomes effective immediately, even if a game is currently running. 

The better solution here would be to add an `int? messageThreadId = null` to the parameter list here, save it in the `Werewolf` game class next to the `ChatId`, and pass it on to this method when appropriate.